RileyLinkKit/UI/RileyLinkDeviceTableViewController.swift

```diff
@@ -27,11 +27,22 @@ public class RileyLinkDeviceTableViewController: UITableViewController, TextFiel
         title = device.name
 
         self.observe()
+        
+        NSTimer.scheduledTimerWithTimeInterval(5, target: self, selector: #selector(RileyLinkDeviceTableViewController.updateRSSI), userInfo: nil, repeats: true)
```

Comment:
If this didn't cause a crash after dismissal, it might means the view controller isn't being deallocated and there's a circular reference somewhere.